teuthology/orchestra/run.py

```diff
@@ -103,7 +110,8 @@ def wait(self):
                 raise CommandCrashedError(command=self.command)
             if status != 0:
```

Comment:
It might make sense to port this feature to the CommandCrashedError and ConnectionLostError. Thoughts?